front/lib/swr/assistants.ts

```diff
@@ -527,6 +527,61 @@ export function useDeleteAgentConfiguration({
   return doDelete;
 }
 
+export function useBatchDeleteAgentConfigurations({
+  owner,
+  agentConfigurationIds,
+}: {
+  owner: LightWorkspaceType;
+  agentConfigurationIds: string[];
+}) {
+  const sendNotification = useSendNotification();
+  const { mutateRegardlessOfQueryParams: mutateAgentConfigurations } =
+    useAgentConfigurations({
+      workspaceId: owner.sId,
+      agentsGetView: "list", // Anything would work
+      disabled: true, // We only use the hook to mutate the cache
+    });
+
+  const doDelete = async () => {
+    if (agentConfigurationIds.length === 0) {
+      return;
+    }
+    const res = await fetch(
+      `/api/w/${owner.sId}/assistant/agent_configurations/delete`,
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          agentConfigurationIds,
+        }),
+      }
+    );
+
+    if (res.ok) {
+      void mutateAgentConfigurations();
+
+      sendNotification({
+        type: "success",
+        title: `Successfully deleted ${agentConfigurationIds.length} agents`,
+        description: `${agentConfigurationIds.length} agents were successfully deleted.`,
```

Comment:
```suggestion
        description: `${agentConfigurationIds.length} agents were successfully archived.`,
```

Also this feels repetitive, how about we say just `Successfully archived` in the title, and show the number in description? 
<img width="338" alt="Screenshot 2025-05-12 at 17 26 13" src="https://github.com/user-attachments/assets/e41d3c88-fb4a-4a9b-badd-280839c8adce" />